dojo/tools/trivy_operator/uniform_vulnid.py

```diff
@@ -0,0 +1,20 @@
+import re
+
+
+class UniformTrivyVulnID:
+    def return_uniformed_vulnid(self, vulnid, test):
```

Comment:
The `test` argument appears unused?

```suggestion
    def return_uniformed_vulnid(self, vulnid):
```